lib/migrator/migrationSteps.js

```diff
@@ -1,28 +1,21 @@
 const path = require('path')
 const lodash = require('lodash')
 const fse = require('fs-extra')
-const { promises: fs } = require('fs')
 const { appDir, projectDir, packageDir } = require('../path-utils')
-const { getConfig } = require('../config')
-const { addReporter, reportSuccess, reportFailure } = require('./reporter')
-const {
-  deleteFile,
-  replaceStartOfFile,
-  removeLineFromFile,
-  deleteDirectory,
-  matchAgainstOldVersions,
-  verboseLog,
-  handleNotFound
-} = require('./fileHelpers')
+const config = require('../config')
+const myReporter = require('./reporter')
+const fileHelpers = require('./fileHelpers')
+// Allows mocking of getOldConfig
+module.exports.getOldConfig = (oldConfigPath) => config.getConfig(require(path.join(projectDir, oldConfigPath)))
 
-const migrateConfig = async () => {
-  const oldConfigPath = path.join(appDir, 'config.js')
+module.exports.migrateConfig = async (oldConfigPath) => {
+  oldConfigPath = path.join(oldConfigPath)
```

Comment:
Why is this needed?

Reply:
Do you mean the path.join?  I think I can remove that but the file is passed in now to match other steps and make it easier to test.

Reply:
Yeah I'm talking about the `path.join` specifically, I'm not sure why you would call `path.join` on just one path part.

Reply:
I've removed the path.join